charts/opentelemetry-ebpf-instrumentation/values.yaml

```diff
@@ -0,0 +1,337 @@
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+#   memory: 128Mi
+
+## -- See `kubectl explain daemonset.spec.updateStrategy` for more
+## ref: https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#daemonset-update-strategy
+updateStrategy:
+  # -- update strategy type
+  type: RollingUpdate
+
+# -- Additional volumes on the output daemonset definition.
+volumes: []
+# - name: foo
+#   secret:
+#     secretName: mysecret
+#     optional: false
+
+# -- Additional volumeMounts on the output daemonset definition.
+volumeMounts: []
+# - name: foo
+#   mountPath: "/etc/foo"
+#   readOnly: true
+
+# -- The nodeSelector field allows user to constrain which nodes your DaemonSet pods are scheduled to based on labels on the node
+nodeSelector: {}
+
+# -- Tolerations allow pods to be scheduled on nodes with specific taints
+tolerations: []
+
+# -- used for scheduling of pods based on affinity rules
+affinity: {}
+
+# -- Adds custom annotations to the Opentelemetry eBPF Instrumentation Pods.
+podAnnotations: {}
+
+# -- Adds custom labels to the Opentelemetry eBPF Instrumentation Pods.
+podLabels: {}
+
+## https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
+# -- Determines how DNS resolution is handled for that pod.
+# If `.Values.preset` is set to `network` or `.Values.config.data.network` is enabled, Opentelemetry eBPF Instrumentation requires `hostNetwork` access, causing cluster service DNS resolution to fail.
+# It is recommended not to change this if Opentelemetry eBPF Instrumentation sends traces and metrics to Grafana components via k8s service.
+dnsPolicy: ClusterFirstWithHostNet
+
+## The below default configuration
+## 1. looks for ALL the services in the host
+## 2. export metrics as prometheus metrics by default at 9090 port
+## 3. enables kubernetes attribute
+## Note: The default configuration is used if config.create=true and config.name=""
+config:
+  # -- set to true, to skip the check around the ConfigMap creation
+  skipConfigMapCheck: false
+  # -- set to true, to use the below default configurations
+  create: true
+  ## -- Provide the name of the external configmap containing the Opentelemetry eBPF Instrumentation configuration.
+  ## To create configmap from configuration file, user can use the below command. Note: The name 'ebpf-instrument-config.yaml' is important.
+  ## `kubectl create cm --from-file=ebpf-instrument-config.yaml=<name-of-config-file> -n <namespace>`
+  ## If empty, default configuration below is used.
+  name: ""
+  # -- default value of Opentelemetry eBPF Instrumentation configuration
+  data:
+    # profile_port: 6060
+    # open_port: 8443
+    # routes:
+    #   unmatched: heuristic
+    # log_level: info
+    ## or alternatively use
+    # grafana:
+    #   otlp:
+    #     cloud_zone: prod-eu-west-0
+    #     cloud_instance_id: 123456
+    #     cloud_api_key:
+    otel_traces_export:
+      endpoint: "http://${HOST_IP}:4317"
```

Comment:
Is this a sensible default? to assume it runs on the same node with an otel agent and send data to there?